src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1669,7 +1693,8 @@
         "taxa": {
           "description": "An array of references used to locate a set of taxonomy reporting descriptors that are always applicable to a result.",
           "type": "array",
-          "minItems": 1,
+          "default": [],
+          "minItems": 0,
```

Comment:
nope. I will now stop providing this comment. minItems == 1, no default.

Reply:
please fix all minItems then merge.

---
In reply to: [270602535](https://github.com/Microsoft/sarif-sdk/pull/1385#discussion_r270602535) [](ancestors = 270602535)